docs/Config.md

```diff
@@ -75,3 +75,45 @@ The secrets in this example are dummy values and won't work in an actual app.
   }
 }
 ```
+
+### Running over HTTPS
+
+You need to change change the `protocol` to `https`, and tell the server where
```

Comment:
Just to expand on that: Hapi is an application server and you need to start up multiple version to listen on different ports, so we'd be starting an extra web server that would just be redirecting traffic to the SSL version (I assume we want  redirect, because the whole point of doing this is so that all traffic goes over https).